qa/core-application-e2e-test-suite/utils/waitForAssertion.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+
+async function waitForAssertion(options: {
```

Comment:
I think this util I created can be replaced by [`toPass`](https://playwright.dev/docs/test-assertions#expecttopass), no?